### PR TITLE
DOC-1432 - RS: Swift is no longer an option for import

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial update

### CODE REVIEW

**Comment by lanceleonard** [comment]
A staged preview [is available](https://docs.redis.com/staging/jira-doc-1432/rs/databases/import-export/import-data/).

File: content/rs/databases/import-export/import-data.md [hunk 1]
```diff
@@ -154,7 +143,7 @@ Before you choose to import from Google Cloud Storage, make sure that you have:
 
 When importing data into an Active-Active database, there are two options:
 
-- Perform a flushall to the database, thus deleting all data.
+- Perform a `flushall` to the database, thus deleting all data.
```
Comment (rrelledge) [inline content/rs/databases/import-export/import-data.md:146]:
```suggestion
- Perform a `flushall` on the database to delete all data.
```

File: content/rs/databases/import-export/import-data.md [hunk 2]
```diff
@@ -28,16 +28,19 @@ Importing data erases all existing content in the database.
...
+1. From the **Import** dialog, enter the details for the import data.  These vary according to the storage location.
+1. To receive email notifications, place a checkmark in the **Receive email notification on success/failure** option.
+1. Select **Import**.
 
 ## Supported storage services
 
-The storage services that are supported for import are:
+You can import data from a variety of services, ranging from local servers to cloud services.
+
+Earlier versions of Redis Enterprise Software supported OpenStack Swift as an storage location; however, that support ended [30 November 2020]({{< relref "/rs/release-notes/rs-5-6-0-april-2020#end-of-life" >}}).  As a result, that option is no longer available.
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/rs/databases/import-export/import-data.md:43]: Fixed.

File: content/rs/databases/import-export/import-data.md (hunk 1 above)
Comment (lanceleonard) [reply content/rs/databases/import-export/import-data.md:146]: Reworked.

**Pull request merged**